omnigibson/scenes/scene_base.py

```diff
@@ -460,15 +461,17 @@ def add_object(self, obj, register=True, _is_call_from_simulator=False):
 
         return prim
 
-    def remove_object(self, obj):
+    def remove_object(self, obj, is_registered=True):
```

Comment:
is this flag necessary? maybe we can just automatically determine if the object is registered or not internally?